projectional/src/main/java/jetbrains/jetpad/projectional/cell/ProjectionalRoleSynchronizer.java

```diff
@@ -28,6 +28,12 @@
   void setCompletion(RoleCompletion<? super ContextT, SourceT> completion);
   void setDeleteHandler(DeleteHandler handler);
   void setClipboardParameters(ContentKind<SourceT> kind, Function<SourceT, SourceT> cloner);
+  <ContentT> void supportContentKind(ContentKind<ContentT> kind,
+      Function<ContentT, SourceT> fromKind,
```

Comment:
Why is it "fromKind"? "fromContent" is more clear I think. The same for "toKind".